components/voice_registry_system.md

```diff
@@ -246,7 +246,31 @@ By following the SRP, we achieve the following: <br>
 #### 1.0.6.1 Do we need central location for common words?
 //todo: #85
 #### 1.0.6.2 Who is the decision maker whether user's utterance is explicit or implicit invocation?
-//todo: #86
+The voice assistant system will decide if a user's utterance is explicit or implicit. For the initial release, VRS will only handle explicit invocations and the voice assistant system will disambiguate implicit invocations.  
+
+**Explicit Invocation Definition:** An invocation of a voice application where the name of the application is part of the invocation utterance. For example:
+- ```Ask Target, to add milk to my shopping list```  
```

Comment:
Per @Jim1151 , add some statement around different from wake word.

Reply:
Updated to include a section defining a wake word and VRS scope:

Note: The name of a voice application is different than the wake word. A wake word is the user's word(s) used to get a voice assistant to start listening. For instance, "Alexa", "Hey Google" and "Hi Bixby" are wake words. If a user says:
``` Alexa, ask Target to add milk to my shopping list ``` then  
``` Alexa``` is the wake word and ```Target``` is the name of the voice application